app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -128,6 +128,72 @@ public function show(Request $request, int $id): JsonResponse
         throw new NotFoundException();
     }
 
+    /**
+     * @OA\Get(
+     *      path="/api/v1/projects/{projectId}/organisations/{organisationId}",
+     *      summary="Get project details by projectID and organisationID",
+     *      description="Fetches project given organisation and project IDs.",
+     *      tags={"Project"},
+     *      @OA\Parameter(
+     *          name="organisationId",
+     *          in="path",
+     *          required=true,
+     *          description="ID of the organisation",
+     *          @OA\Schema(type="integer")
+     *      ),
+     *      @OA\Parameter(
+     *          name="projectId",
+     *          in="path",
+     *          required=true,
+     *          description="ID of the project",
+     *          @OA\Schema(type="integer")
+     *      ),
+     *      @OA\Response(
+     *          response=200,
+     *          description="Successfully retrieved project",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="success"),
+     *              @OA\Property(property="data", type="array",
+     *                  @OA\Items(
+     *                      ref="#/components/schemas/User"
+     *                  )
+     *              )
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=404,
+     *          description="Project not found",
+     *          @OA\JsonContent(
+     *              type="object",
+     *              @OA\Property(property="message", type="string", example="Project not found")
+     *          )
+     *      )
+     * )
+     */
+    public function getProjectByIdAndOrganisationId(Request $request, int $projectId, int $organisationId): JsonResponse
+    {
+        $project = Project::with([
+                'projectDetail', 
+                'custodians', 
+                'modelState.state',
+                'custodianHasProjectOrganisation' => function($query) use ($organisationId) {
+                    $query->whereHas('projectOrganisation', function($query2) use ($organisationId) {
+                            $query2->where('organisation_id', $organisationId);
+                        })
+                    ->with('modelState.state');
+                },
+            ])->findOrFail($projectId);
+
+        if ($project) {
+            return response()->json([
+                'message' => 'success',
```

Comment:
We've got a Responses trait that handles all this that the FE is prepared for already